inference-engine/cmake/config.cmake.in

```diff
@@ -4,6 +4,10 @@
 
 if(DEFINED IE_MAIN_SOURCE_DIR AND TARGET inference_engine)
     set(InferenceEngine_LIBRARIES inference_engine inference_engine_c_api)
+    add_library(IE::inference_engine ALIAS inference_engine)
+    if(TARGET inference_engine_c_api)
+        add_library(IE::inference_engine_c_api ALIAS inference_engine_c_api)
```

Comment:
should not we check that target is already defined? E.g.
```cmake
find_package(InferenceEngine)
find_package(InferenceEngine)
```
should work

Reply:
Done